homeassistant/components/media_player/plex.py

```diff
@@ -457,6 +474,13 @@ def force_idle(self):
         self._session = None
         self._clear_media_details()
 
+    async def async_added_to_hass(self):
+        """Callback when entity is added to hass."""
+        if PLEX_DATA not in self.hass.data:
+            self.hass.data[PLEX_DATA] = PlexData()
+        if self.machine_identifier not in self.hass.data[PLEX_DATA].devices:
```

Comment:
This is also not possible. Home Assistant will have already raised if an entity had a duplicate unique id.

Reply:
You need to check this BEFORE calling `add_devices([entity])`